pywr/_core.pyx

```diff
@@ -500,7 +500,7 @@ cdef class AggregatedNode(AbstractNode):
             self.model.dirty = True
 
     cpdef after(self, Timestep ts):
-        AbstractStorage.after(self, ts)
+        AbstractNode.after(self, ts)
```

Comment:
I think all the work was in `Storage.after` rather than `AbstractStorage.after`.